Task11/Task11/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+namespace Table
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите число строк таблицы");
+            var m = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите число столбцов таблицы");
+            var n = int.Parse(Console.ReadLine());
+
+            var check = true;
+            while (check)
+            {
+                check = (m <= 5 || m >= 20 || n <= 5 || n >= 20);
```

Comment:
Здесь неравенства должны быть строгими (размер от 5 до 20 включительно!)
```suggestion
                check = (m < 5 || m > 20 || n < 5 || n > 20);
```